python/metatensor-torch/tests/atomistic/model.py

```diff
@@ -270,6 +271,26 @@ def test_access_module(tmpdir):
     loaded_atomistic.module.other
 
 
+def test_is_atomistic_model():
+    model = FullModel()
+    model.train(False)
+
+    capabilities = ModelCapabilities(
+        interaction_range=0.0,
+        supported_devices=["cpu"],
+        dtype="float64",
+    )
+    atomistic = MetatensorAtomisticModel(model, ModelMetadata(), capabilities)
+    scripted_atomistic = torch.jit.script(atomistic)
```

Comment:
Here I think we should also test that it works with `.save()` and then loading it again with `metatensor.torch.load_atomistic_model` (which is the way we're supposed to handle these objects)